conductor/src/test/java/com/bluelinelabs/conductor/util/AttachFakingFrameLayout.java

```diff
@@ -77,6 +79,8 @@ public boolean unlinkToDeath(DeathRecipient recipient, int flags) {
     };
 
     private boolean reportAttached;
+    private boolean needDelayPost;
+    private List<Runnable> delayedPosts = new ArrayList<>();
```

Comment:
Probably a good place to use `LinkedList` instead of` ArrayList`, so it allocates less place for storing stuff and could be used as a queue.

Reply:
Why you think so, we don't use a queue in this place. 
In most of the use cases, ArrayList prefers than LinkedList.
This is test code, so there is no big deal